01-node-tutorial/answers/writeWithPromisesAwait.js

```diff
@@ -0,0 +1,31 @@
+const {writeFile, readFile } = require("fs").promises;
+
+async function writer() {
+   try{
+    await writeFile('./temporary/temp.txt','Hello world/n This is my second line/n This is my third line' )
```

Comment:
Requires changes:
I believe the assignment wanted you to write to the file 3 times, instead of using `\n`. This would resemble the `writeWithPromisesThen` example, where you write to the same file 3 times as well.
```js
await writeFile("...", "...");
await writeFile("...", "...", { flag: "a" });
await writeFile("...", "...", { flag: "a" });
```